### PR TITLE
feat: Tweak of the Motivation section of `0000-cargo-templates.md`.

### PR DESCRIPTION
**Pull request by ckaran**
@jsjoeio If you are willing to accept MRs to your RFC and to manage the RFC, I can help out by sending you MRs based on the discussions we've had.  However, I'm still just the babysitter!  You're the parent and need to take care of this baby! :laughing: 

Are you willing to do that?  It'll be a little less work for you, but you'll still need to keep pushing it through the process.

### CODE REVIEW

**Comment by jsjoeio** [comment]
@ckaran thanks for the tweak! i mean I'm happy to accept MRs to the RFC and check notifications here and there, but again, I'd love to pass it off to someone since I no longer have interest in this (the whole thing has diverged from why I original wanted to do this 😜)

I think I told you (maybe not). The only reason I got involved in this whole thing was because when I was learning rust and using `cargo new` I thought, "wouldn't it be great if the basic template had a `README`?"

And yes, then that made me go down this rabbit hole. 

Long story short, I will still get notifications and do what I can, but I hope if this gets approved someone else can drive!

**Comment by ckaran** [comment]
I fully understand, and I do remember that you mentioned that you can't keep pushing on it.  The one good thing is that if this is approved in the way I expect, it will just be a documentation update; no pushing on code.  So, it will just involve a lot of typing.

**Comment by jsjoeio** [comment]
> The one good thing is that if this is approved in the way I expect, it will just be a documentation update; no pushing on code. So, it will just involve a lot of typing.

🤞🏼 let's hope you're right!
